src/main/java/com/dev/cinema/model/dto/movie/MovieRequestDto.java

```diff
@@ -1,9 +1,13 @@
 package com.dev.cinema.model.dto.movie;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
 import lombok.Data;
 
 @Data
 public class MovieRequestDto {
+    @NotNull
```

Comment:
ideally it's better to add some informative message

Reply:
done